aiopg/extras.py

```diff
@@ -0,0 +1,43 @@
+def _paginate(seq, page_size):
```

Comment:
```
import itertools 


def _paginate(seq, page_size):
    it = iter(seq)
    page = list(itertools.islice(it, 0, page_size))
    while page:
        yield page
        page = list(itertools.islice(it, 0, page_size))
```
I think it is better solution

Reply:
with this solution you can you iterators in _paginate
for example 
`_paginate(range(1, 8), 3)`